animation-events/js/controls.js

```diff
@@ -0,0 +1,107 @@
+import {Component, Property} from '@wonderlandengine/api';
+import { GunController } from './gun-controller.js';
+
+/**
+ * Controls
+ *
+ * Responsible for reading input and forwarding it to the gun controller.
+ */
+export class Controls extends Component {
+    static TypeName = 'controls';
+
+    static Properties = {
+        shootMouseButton: Property.int(0),
+        reloadButton: Property.int(82),
+        drawFirstWeaponButton: Property.int(49),
+        drawSecondWeaponButton: Property.int(50),
+        holsterWeaponButton: Property.int(91)
```

Comment:
These might need a bit of comments, maybe referencing the keyCode documentation on MDN or something.

(Also which keys they are referring to with the default values).

Optionally, feel free to remove them and hard-code further down (with comment on which keys they respond to) to simplify the example a bit.